dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/internal/LibertyLoader.java

```diff
@@ -30,27 +31,23 @@ public abstract class LibertyLoader extends SecureClassLoader implements NoClass
 
     final ClassLoader parent;
 
+    // This is used to help when analyzing ClassLoader leaks to show which ones are active and which ones were destroyed.
+    volatile boolean isDestroyed = false;
```

Comment:
Should it also be used to log errors if someone attempts to call loadClass/findResource etc. on a stale loader?

Reply:
Yeah I am going to have to undo the destroy parts of this.  At least the calls to destroy since it can cause zero migration issues where application operations can fail.  I can move the destroy to later which is needed anyway to do it after the application state listeners stopped is called.  There is a failing test in the personal build that demonstrates the problem so to be safe I am going to remove the call to destroy and work on another solution in a follow on PR.  The loadClass is failing already because the destroy methods in the sub classes are removing references to the Containers that have the actual classes.